### ISSUE TITLE
Error when deserialising BatchNormalization layers from yaml #37885

### ISSUE THREAD

**Issue by kimheesung-samsung** [kimheesung-samsung/sos-masters#2]
https://github.com/tensorflow/tensorflow/issues/37885
